### ISSUE TITLE
setup-ruby doesn't work on ubuntu-20.04

### ISSUE THREAD

**Issue by jwillemsen** [actions/setup-ruby#70]
ubuntu-20.04 is available (see https://github.com/actions/virtual-environments#available-environments), but when using setup-ruby we get `##[error]Version 2.7 not found`. I created the PR https://github.com/actions/setup-ruby/pull/69 to demonstrate this, currently ubuntu-latest points to ubuntu-18.04 so this is not detected by the default testing

**Comment by jwillemsen** [comment]
It does work when using 2.7.x as version

**Comment by jwillemsen** [comment]
Looks with the released action this doesn't work yet, see for example https://github.com/RemedyIT/axcioma/pull/49/checks?check_run_id=753730607

**Comment by MSP-Greg** [comment]
I've gotten several emails this morning showing your work, not quite sure what your goal is.

Re Ubuntu 20.04 & Ruby, all versions are available when using [ruby/setup-ruby](https://github.com/ruby/setup-ruby)

**Comment by jwillemsen** [comment]
I wanted to upgrade one of our github actions usage to use 20.04. Our action script uses actions/setup-ruby and this doesn't seem to work with 20.04 correctly

**Comment by eregon** [comment]
I'd suggest to use [ruby/setup-ruby](https://github.com/ruby/setup-ruby) which has all versions and 20.04 support since last weekend.
`ruby/setup-ruby` is BTW now used in the Ruby starter workflow.

**Comment by jwillemsen** [comment]
Ok, will try to use ruby/setup-ruby

**Comment by MSP-Greg** [comment]
@jwillemsen

Sorry, I wasn't clear (not enough coffee yet).  ruby/setup-ruby is used by almost all of the repos within the Ruby org, and many other popular gems.  It makes multi-platform CI with Ruby as easy as possible...

**Comment by jwillemsen** [comment]
Yes, switching to it right now. Did make a PR for ruby/setup-ruby to test with all virtual environments explicitly to make sure everything works

**Comment by mondeja** [comment]
I see that Bundler is not included using `ruby-version: 2.5` so I suggest #80
